src/app/3d/qgs3dnavigationwidget.cpp

```diff
@@ -186,6 +188,43 @@ Qgs3DNavigationWidget::Qgs3DNavigationWidget( Qgs3DMapCanvas *parent ) : QWidget
   }
   );
 
+  mCameraInfo = new QTableView( this );
+  mCameraInfo->setEditTriggers( QAbstractItemView::NoEditTriggers );
+
+  mCameraInfoItemModel = new QStandardItemModel( this );
+
+  mCameraInfoItemModel->appendRow( QList<QStandardItem *> { new QStandardItem, new QStandardItem } );
+  mCameraInfoItemModel->appendRow( QList<QStandardItem *> { new QStandardItem, new QStandardItem } );
+  mCameraInfoItemModel->appendRow( QList<QStandardItem *> { new QStandardItem, new QStandardItem } );
+  mCameraInfoItemModel->appendRow( QList<QStandardItem *> { new QStandardItem, new QStandardItem } );
+  mCameraInfoItemModel->appendRow( QList<QStandardItem *> { new QStandardItem, new QStandardItem } );
+  mCameraInfoItemModel->appendRow( QList<QStandardItem *> { new QStandardItem, new QStandardItem } );
+  mCameraInfoItemModel->appendRow( QList<QStandardItem *> { new QStandardItem, new QStandardItem } );
+  mCameraInfoItemModel->appendRow( QList<QStandardItem *> { new QStandardItem, new QStandardItem } );
+
+  mCameraInfoItemModel->setData( mCameraInfoItemModel->index( 0, 0 ), QStringLiteral( "Near plane" ) );
+  mCameraInfoItemModel->setData( mCameraInfoItemModel->index( 1, 0 ), QStringLiteral( "Far plane" ) );
+  mCameraInfoItemModel->setData( mCameraInfoItemModel->index( 2, 0 ), QStringLiteral( "Camera X pos" ) );
+  mCameraInfoItemModel->setData( mCameraInfoItemModel->index( 3, 0 ), QStringLiteral( "Camera Y pos" ) );
+  mCameraInfoItemModel->setData( mCameraInfoItemModel->index( 4, 0 ), QStringLiteral( "Camera Z pos" ) );
+  mCameraInfoItemModel->setData( mCameraInfoItemModel->index( 5, 0 ), QStringLiteral( "Looking at X" ) );
+  mCameraInfoItemModel->setData( mCameraInfoItemModel->index( 6, 0 ), QStringLiteral( "Looking at Y" ) );
+  mCameraInfoItemModel->setData( mCameraInfoItemModel->index( 7, 0 ), QStringLiteral( "Looking at Z" ) );
```

Comment:
You can use QStandardItem constructor in appendRow calls that takes item text to avoid these setData calls:
`new QStandardItem("Near plane")`